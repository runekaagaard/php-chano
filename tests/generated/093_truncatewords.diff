001- 123
001+ Fatal error: Call to undefined method DtlIter::truncatewords() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/093_truncatewords.php on line 4