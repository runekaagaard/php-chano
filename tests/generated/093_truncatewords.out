Fatal error: Call to undefined method DtlIter::truncatewords() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/093_truncatewords.php on line 4